pkcs7.go

```diff
@@ -875,10 +884,38 @@ func encryptAES128GCM(content []byte) ([]byte, *encryptedContentInfo, error) {
 	return key, &eci, nil
 }
 
-func encryptDESCBC(content []byte) ([]byte, *encryptedContentInfo, error) {
-	// Create DES key & CBC IV
-	key := make([]byte, 8)
-	iv := make([]byte, des.BlockSize)
+func encrypt(content []byte, contentEncryptionAlgorithm int) ([]byte, *encryptedContentInfo, error) {
+	var keySize int
```

Comment:
you can group all vars with a ()

```
var (
    foo int
    bar string
)
```

Just a style nit.

Reply:
Fixed up in fa1b68b